AndelaOSP/art-dashboard
# #160312080 Keep filters options checked after applying

## What does this PR do?
Enables filters to stay checked even after applying them

## Description of Task to be completed?
- Create action and reducer
- Refactor components and add functionality
- Refactor tests
- Add css to reduce filters font size

## How should this be manually tested?
- Head on to the `asset lists` page. Click on the `filter` button. Check a couple of filter options, then click `apply filters`

## What are the relevant pivotal tracker stories?
[#160312080](https://www.pivotaltracker.com/n/projects/2146417/stories/160312080)

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
- Clicking on the side of the `filter` button closes the dropdown as well as clicking on `apply filters`
![](https://dzwonsemrish7.cloudfront.net/items/19472M2l371X2p2z3x41/Screen%20Recording%202018-09-14%20at%2005.25%20PM.gif)

- Filter options stay checked after applying. They have also been reduced in size.

![](https://dzwonsemrish7.cloudfront.net/items/242h3q3Z1Q1E1U2R2M1l/Screen%20Recording%202018-09-14%20at%2005.28%20PM.gif)
